lropt/robust_problem.py

```diff
@@ -343,6 +293,321 @@ def _construct_chain(
         # return a chain instead (chain.apply, return the problem and inverse data)
         return SolvingChain(reductions=new_reductions)
 
+    #TODO (bart): please let's avoid camel case and use snake case instead
+    def _isMROSet(self, unc_set):
+        '''
+        This function returns whether we work on MRO set or not.
+
+        Arguments
+        ---------
+        unc_set
+            Uncertainty set
+        
+        Returns
+        -------
+            Boolean result
+        '''
+
+        return (type(unc_set)==MRO)
+
+    def _setNewprob(self, override=False):
+        '''
+        This function cannonizes a problem and saves it to self.new_prob
+
+        Arguments
+        ---------
+        override 
+            If True, will override current new_prob. If false and new_prob exists, does nothing.
+
+        returns
+        -------
+        None
+        '''
+
+        if (not override) and (self.new_prob):
+            return
+
+        #Creating uncertainty reduction and chain
+        unc_reductions = []
+        if type(self.objective) == Maximize:
+            unc_reductions += [FlipObjective()]
+        unc_reductions += [RemoveUncertainParameters()]
+        newchain = UncertainChain(self, reductions=unc_reductions)
+
+        # TODO (bart): We should keep inverse data. We can use it to reconstruct the original
```

Comment:
I have added this to the code comment as well. I think we need inverse data. How can we return the original solution and objective otherwise?

Reply:
This variable is not used in the code, hence not assigned. The function still returns it, should we want to use it in the future.